pages/api/index.js

```diff
@@ -19,8 +19,8 @@ export default async function handler(req, res) {
     })
   )
   await page.setViewport({
-    width: 1920,
-    height: 1080,
+    width: parseInt(width) || 1920,
```

Comment:
I'd move the var declaration into the destructure. Makes it easier to know there is a default.

Reply:
Seems like I can't push to the branch anymore, @jeffpatzer can you update the file to?

`var minify = require('html-minifier').minify
import { generatePage } from '@/lib/page'
import chromium from 'chrome-aws-lambda'

export default async function handler(req, res) {
  const { title, width = 1920, height = 1080 } = req.query
  const dom = generatePage(title)
  const browser = await chromium.puppeteer.launch({
    args: chromium.args,
    defaultViewport: chromium.defaultViewport,
    executablePath: await chromium.executablePath,
    headless: chromium.headless,
    ignoreHTTPSErrors: true,
  })
  const page = await browser.newPage()
  await page.setContent(
    minify(dom, {
      collapseWhitespace: true,
    })
  )
  await page.setViewport({ width: parseInt(width), height: parseInt(height) })
  const content = await page.$('body')
  const imageBuffer = await content.screenshot({ omitBackground: true })
  await page.close()
  await browser.close()
  res.setHeader(
    'Cache-Control',
    `public, immutable, no-transform, s-maxage=31536000, max-age=31536000`
  )
  res.setHeader('Content-Type', 'image/png')
  res.send(imageBuffer)
  res.status(200)
}
`